graphql-schemas/schema.yml

```diff
@@ -924,6 +924,19 @@ confs:
       schema: /aws/policy-1.yml
       subAttr: account
 
+- name: CloudflareAccount_v1
+  interface: ExternalResourcesProvisioner_v1
+  fields:
+  - { name: schema, type: string, isRequired: true }
+  - { name: path, type: string, isRequired: true }
+  - { name: name, type: string, isRequired: true, isUnique: true, isSearchable: true }
+  - { name: description, type: string }
+  - { name: providerVersion, type: string, isRequired: true }
+  - { name: accountOwners, type: Owner_v1, isList: true, isRequired: true }
+  - { name: enableDeletion, type: boolean }
+  - { name: apiCredentials, type: VaultSecret_v1, isRequired: true }
+  - { name: terraformStateAccount, type: AWSAccount_v1, isRequired: true }
```

Comment:
I know early on we had a lot of discussions about what this would look like. I thought that we landed on `automationToken` and doing things much like the AWS account state is handled. Can you just share what pushed you in this direction? I think it's a minor point for now as long as the keys are handled properly (no chance of overlapping keys, which looks like is the case from q-r code).

Reply:
I chose to link to an AWS account (and named the key terraformStateAccount) because the integration needs both the `automationToken` and `terraformState` from the AWS account. I could have had 2 fields that links to either instead but having just 1 feels right and I cannot think of a case where we would need either from different accounts